apps/qubit/modules/digitalobject/actions/metadataComponent.class.php

```diff
@@ -307,9 +307,13 @@ protected function setOriginalFileShowProperties()
       check_field_visibility('app_element_visibility_digital_object_preservation_system_original_file_size')
       && !$this->isEmpty($this->resource->object->originalFileSize)
     );
+    // Convert this UTC string property to local time
+    $this->originalFileIngestedAt = $this->localizeUTCDatetime(
+      (string)$this->resource->object->originalFileIngestedAt
```

Comment:
Most examples of PHP type casting I've seen have a space between the cast and the variable, including in the PHP docs:
https://www.php.net/manual/en/language.types.type-juggling.php#language.types.typecasting

I'd like to standardize on this convention for AtoM

Reply:
:+1: I also updated another existing type cast in the same template.